plugins/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/requirement/SecureStorageRequirement.java

```diff
@@ -0,0 +1,95 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.cdk.reddeer.requirement;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.List;
+
+import org.jboss.reddeer.common.exception.WaitTimeoutExpiredException;
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.common.wait.TimePeriod;
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.reddeer.core.condition.JobIsRunning;
+import org.jboss.reddeer.core.condition.WidgetIsFound;
+import org.jboss.reddeer.core.matcher.ClassMatcher;
+import org.jboss.reddeer.core.matcher.WithMnemonicTextMatcher;
+import org.jboss.reddeer.eclipse.equinox.security.ui.StoragePreferencePage;
+import org.jboss.reddeer.junit.requirement.Requirement;
+import org.jboss.reddeer.swt.api.TableItem;
+import org.jboss.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.jboss.tools.cdk.reddeer.requirement.SecureStorageRequirement.SecureStorageDisabled;
+
+/**
+ * Requirement class for switching all requests from Secure Storage preferences off
+ * @author odockal
+ *
+ */
+public class SecureStorageRequirement implements Requirement<SecureStorageDisabled> {
```

Comment:
I think that better place for this class would be o.j.t.common.reddeer so other plugins can use this requirement without dependency to o.j.t.cdk.reddeer plugin. Or maybe put it into reddeer? @rawagner